code-challenges.js

```diff
@@ -21,41 +21,135 @@ const fruit4 = "kiwi"
 
 // Pseudo code:
 
+// Use provide variables. 
+// Dcelcare a function expression called boilingPoint.
+// Set parameters (degrees1, degrees2, degrees3).
+// Use conditional statement to compare passed-in strings:
+    //if degrees1 is less than or equal to 42 -- we will return "42 is below boiling point"
+    //if degrees2 is equal to 212 -- we will return "212 is at boiling point"
+    //if degree3 is greater than or equal to 350 -- we will return "350 is at boiling point"
+    //use else or else if to evaluate non-numerical digits and return the other with "Invalid entry, temperature not possible in this reality."
+
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const boilingPoint = (degree1, degree2, degree3) => {
+    if(degree1 <= 42) {
+        return "42 is below boiling point"
+    } else if (degree2 = 212) {
+        return "212 is at boiling point"   
+    } else if (degree3 > 350) {
+        return "350 is at boiling point"
+    }
+}
+    console.log (boilingPoint(2)) // 42 is below boiling point
+    console.log (boilingPoint(36)) // 42 is below boiling point
+    console.log (boilingPoint(212)) // 212 is at boiling point
+    console.log (boilingPoint(500)) // 212 is at boiling point 
+    
+    // Note to Instructor: I'm not sure why "degree3" parameters aren't showing up at "350 is at boiling point". I've tried line #42 with the >= and > but in both cases as well even attempted to create a seperate/additional solo "if" statement just for that line of code but regardless I was unable to get the desired output. Are there only supposed to be 2 parameters or is 3 allowed? At least one of my parameters doesn't seem to be registersing on some level as I keep getting this synthax error "'degree2' (*Sometimes its 'degree3') is declared but its value is never read." I found this odd/confusing. 
```

Comment:
Steven - it looks like you put a lot of effort into this problem.  Thank you for your great explanation and request for feedback.  A couple of things to note:  we want this to be dynamic so you can pass any number in and get a response, so you don't want to hardcode outputs.  Also, the reason you were running into issues is becuase on line 40, you use the assignment (=) instead of the strict (===) operator.  That will make weird things happen (as you saw!).